libs/platform/src/lib/components/approval-flow/approval-flow.component.ts

```diff
@@ -261,6 +264,17 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
     /** @hidden */
     ngOnDestroy(): void {
         this.subscriptions.unsubscribe();
+        this._onDestroy$.complete();
```

Comment:
is this going to work without `this._onDestroy$.next();`?

Reply:
also it'd be better not to mix approaches for unsubs and pick only one, now `_onDestroy ` is used along with `this.subscriptions`

Reply:
using this.subscriptions now.